astropy/table/tests/test_table.py

```diff
@@ -2392,3 +2392,15 @@ class MyTable(table.Table):
         assert type(t['c']) is MyMaskedColumn
         assert type(t['d']) is MySubColumn
         assert type(t['e']) is MySubMaskedColumn  # sub-class not downgraded
+
+
+def test_iterrows():
+    t = table.Table([(2, 3, 2, 1), (8, 7, 6, 5)], names=('a', 'b'))
+    assert t.iterrows(['a']) == zip(t['a'])
+    assert t.iterrows(['a', 'b']) == zip(t['a'], t['b'])
+    with pytest.raises(ValueError) as err:
+        t.iterrows()
+        err.value.args[0] == 'No input Provided'
```

Comment:
This needs to be de-dented by 4 spaces (even with the `with pytest...`) because the way it is written the `err.value` line is never reached because of the exception in `t.iterrows()`.  Same below.